doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -601,6 +601,8 @@ Bluetooth libraries and services
     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN` Kconfig option to the :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_PN` Kconfig option.
     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX` Kconfig option to the :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_PN_LEN_MAX` Kconfig option.
 
+  * Changed the Fast Pair storage module to overwrite the least recently used Account Key instead of the oldest Account Key on Account Key write.
```

Comment:
```suggestion
  * Updated the Fast Pair storage module to overwrite the least recently used Account Key instead of the oldest Account Key on Account Key write.
```